book/chapters/chapter2/data_and_basic_modeling.qmd

```diff
@@ -562,7 +563,7 @@ as.data.table(msr())
 All measures implemented in `mlr3` are defined primarily by three components: 1) the function that defines the measure; 2) whether a lower or higher value is considered 'good'; and 3) the range of possible values the measure can take.
 As well as these defining elements, other metadata are important to consider when selecting and using a `Measure`, including if the measure has any special properties (e.g., requires training data), the type of predictions the measure can evaluate, and whether the measure has any 'control parameters'.
 All this information is encapsulated in the `r ref("Measure", aside = TRUE)` object.
-By example, let us consider the `r index('mean absolute error', parent = "measures")` (MAE):
+For example, let us consider the `r index('mean absolute error', parent = "measures")` (MAE):
```

Comment:
By example is correct. We are demonstrating something by an example

Reply:
Yes, but it sounds weird.

Reply:
Sure but it's grammatically correct and we've chosen formal writing everywhere